AK/Time.h

```diff
@@ -66,14 +66,44 @@ constexpr int days_in_year(int year)
     return 365 + (is_leap_year(year) ? 1 : 0);
 }
 
+namespace Detail {
+// Consider the integers of the interval [begin, end), and a modulus 'mod'.
+// For each number 'n' in the interval, n % mod may or may not be 0.
+// This function counts how many such 'n' there are in the given interval.
```

Comment:
"may or may not be 0", "how many such" – not terribly clear from the comment what's going on. Does it count those that may be 0, may not be 0, both?

Either reword ("Counts how many numbers n are in [begin, end) with n % positive_mod == 0"), or remove comment.

Reply:
I love your version, thanks!